csharp/private/providers.bzl

```diff
@@ -102,9 +110,13 @@ FrameworkCompatibility = {
     "netstandard2.1": ["netstandard2.0"],
 
     # .NET Framework
+    "net11": [],
     "net20": [],
-    "net40": ["net20"],
-    "net45": ["net40", "netstandard1.1"],
+    "net30": [],
+    "net35": [],
```

Comment:
Should we just fill out these edges? As you've shown, we're technically supporting building with them, just without the availability of the framework.

Does not having `"net35": ["net30"]` mean you can't build a net45 library, that depends on a net35 nuget, which itself depends on a net30 nuget?

Reply:
I was freaked out about accidentally creating `@net35//:System.Linq`. I don't feel strongly about it though.

Yeah it might. Ok I'll do the normal thing.